### PR TITLE
494: Adds rollup to create umd bundle and packages for angular-cli.

### PR DESCRIPTION
**Pull request by glawson**

### CODE REVIEW

File: dist/bossy.umd.js [hunk 1]
```diff
@@ -0,0 +1,526 @@
+(function (global, factory) {
```
Comment (tabersays) [inline dist/bossy.umd.js]: Did you mean to commit this to dist

File: dist/bossy.umd.min.js [hunk 2]
```diff
@@ -0,0 +1 @@
...anger=s}return t}(),I=function(){function t(t){var e=this;Object.keys(t).forEach(function(o){e[o]=t[o]})}return t}(),D=function(){function t(t,e,o){this.items=t,this.isInline=e,this.labelId=o}return t}(),v=function(){function t(t,e,o,n,s,r,i,a){this.name=t,this.type=e,this.items=o,this.isSplit=n,this.size=s,this.isDropup=r,this.isRightAligned=i,this.variant=a}return t}(),b=function(){function t(t,e,o,n){this.type=t,this.name=e,this.href=o,this.isDisabled=n}return t}(),M=function(){function t(){}return t.forRoot=function(){return{ngModule:t,providers:[]}},t}();M.decorators=[{type:e.NgModule,args:[{imports:[o.BrowserModule,n.ReactiveFormsModule],declarations:[s,r,i,a,l,u,p,c,d,h],exports:[s,r,i,l,u,p,d,h]}]}],M.ctorParameters=function(){return[]},t.BossyModule=M,t.BossyCalendar=s,t.BossyCalendarConfig=f,t.BossyForm=i,t.BossyFormConfig=m,t.BossyFormInput=r,t.BossyFormInputConfig=y,t.BossyFormLabel=a,t.BossyFormLabelConfig=g,t.BossyFormSelectMenu=d,t.BossyFormSelectMenuConfig=I,t.BossyFormInputError=c,t.BossyRadio=l,t.BossyRadioConfig=D,t.BossyDropdown=u,t.BossyDropdownConfig=v,t.BossyDropdownMenuItem=p,t.BossyDropdownMenuItemConfig=b,Object.defineProperty(t,"__esModule",{value:!0})});
```
Comment (tabersays) [inline dist/bossy.umd.min.js]: Did you mean to commit this to dist

File: gulpfile.js [hunk 3]
```diff
@@ -6,31 +6,31 @@ const shell = require('gulp-shell');
...
-	gulp.src([
-		'./dist/components/**/*.js',
-		'./dist/directives/**/*.js'
-	], {read: false})
-		.pipe(shell([
-			//'echo <%= file.path.replace(file.cwd + "/dist/", file.cwd + "/dist/jsdoc/") %>'
-			'node node_modules/jsdoc/jsdoc.js <%= file.path %> -c jsdoc.json > <%= file.path.replace(file.cwd + "/dist/", file.cwd + "/dist/jsdoc/") %>'
-		]));
-});
+// gulp.task('build-doc', function () {;
```
Comment (tabersays) [inline gulpfile.js]: Did you mean to comment this out, if so remove it.

File: src/index.ts [hunk 4]
```diff
@@ -12,6 +12,29 @@ import {BossyFormInputError} from './components/form-input-error';
 import {BossyFormSelectMenu} from './components/form-selectmenu';
 import {BossyFormTextarea} from './components/bossy-form-textarea';
 
+//export {BossyCalendarModule} from './calendar/calendar-module';
```
Comment (tabersays) [inline src/index.ts]: Remove unneeded code.

File: src/index.ts [hunk 5]
```diff
@@ -12,6 +12,29 @@ import {BossyFormInputError} from './components/form-input-error';
...
+export {BossyCalendar} from './components/calendar';
+export {BossyCalendarConfig} from './config/calendar';
+
+//export {BossyFormModule} from './form/form-module';
```
Comment (tabersays) [inline src/index.ts]: Remove unneeded code.

File: dist/bossy.umd.js (hunk 1 above)
Comment (glawson) [inline dist/bossy.umd.js]: Yes - until I have it building on commit.

File: dist/bossy.umd.min.js (hunk 2 above)
Comment (glawson) [inline dist/bossy.umd.min.js]: Yes - until I have it building on commit.

**Comment by glawson** [comment]
@tabersays - sandbox and tests work again

**Comment by glawson** [comment]
@tabersays - merged code renamed stuff, fixed.

**Pull request merged**